src/frameworks.sources

```diff
@@ -1952,6 +1953,7 @@ IOS_FRAMEWORKS =         \
 	$(COMMON_FRAMEWORKS) \
 	AVKit \
 	Accounts \
+	AdServices \
```

Comment:
Does `AdServices` exist for Mac Catalyst as well?

Reply:
Yes according to docs [Mac Catalyst 14.3+](https://developer.apple.com/documentation/AdServices?language=objc)

Reply:
Then you need an entry in the list of Mac Catalyst frameworks in this file (`MACCATALYST_FRAMEWORKS`).

Reply:
@rolfbjarne done, thank you!